wordssite/templates/dictionary/translation_confirm_delete.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+{% url 'dictionary:all' as list_all %}
+
+<div class="container d-flex justify-content-center mt-5" style="width: 50%;">
+    <div class="col mt-5 text-center">
+        <h1 class="mb-5">Delete Translation</h1>
+
+        <p class="h5">Are you sure you want to delete the Translation:</p>
+        <h3 class="my-5">{{translation.translation}}</h3>
+        <form action="" method="POST">
+            {% csrf_token %}
+            <input type="submit" class="btn btn-primary btn-lg" value="Confirm" style="margin-right: 30px; margin-top: 30px;">
```

Comment:
I'd go with bootstrap margins: https://getbootstrap.com/docs/4.0/utilities/spacing/, so it's all bootstrap,
but that's a minor comment.

Reply:
Noted for the UI branch

Reply:
https://ido-rot.monday.com/boards/2588079332/pulses/2826144538